subt/tf_detector.py

```diff
@@ -0,0 +1,121 @@
+"""Artefacts detection via tensorflow"""
+
+import os
+import tensorflow as tf
+import numpy as np
+import cv2
+
+# Directory with files saved_model.pb and saved_model.pbtxt
```

Comment:
can we have the model on `osgar.robotika.cz` or it is already there?

Reply:
Yes, that would be more useful. Can you upload it? i have no access.
I am not sure how it can be implemented then.